src/Builder/ImportONNXUtils.cpp

```diff
@@ -0,0 +1,130 @@
+#include "ImportONNXUtils.hpp"
+#include <map>
+#include <set>
+#include <vector>
+
+bool TopologicallySorted(const onnx::GraphProto &graph) {
```

Comment:
nit: if named "IsTopologicallySorted" would make the result of the function more obvious.

Reply:
Thats a good point. I've updated it accordingly